test/cardTests/creditCardTests.java

```diff
@@ -1087,4 +1087,7 @@ void testUpdateCreditCardInformationUpdateBillingAddress() {
 			fail("Error inserting test card (Database failure)");
 		}
 	}
+	
+	
```

Comment:
did you make a test for it in creditCardTests, add a test specifically for the static method you made

Reply:
also leave test for adding card with existing nickname but just make a new one too that only tests the static method